### PR TITLE
Deployment improvements

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

- Ensures pre and post scripts are working  
- Changes Sentry plugin to only upload sourcemaps if token is given
- Disables caching on index files  
- Adds route that removes index.html from url

### Preview

https://beta.electricitymaps.com/map/index.html

### CODE REVIEW

File: web/src/features/panels/LeftPanel.tsx [hunk 1]
```diff
@@ -13,10 +14,28 @@ import {
...
+        to={{
+          pathname: pathWithoutIndex,
+        }}
+      />
+    );
+  }
+
+  return children;
+}
+
```
Comment (VIKTORVAV99) [inline web/src/features/panels/LeftPanel.tsx:38]: There must be a simpler way to do this by using wildcard routes and internal functions. But lets keep this for now and see if we can improve this after it's launched.

File: web/src/features/modals/InfoModal.tsx [hunk 2]
```diff
@@ -81,6 +81,7 @@ export function InfoModalContent() {
           {__('info-modal.legal-notice')}
         </a>
       </div>
+      <p className="text mt-2  text-sm">Version: {APP_VERSION}</p>
```
Comment (tonypls) [inline web/src/features/modals/InfoModal.tsx:84]: Very useful 😎

**Review by tonypls — APPROVED** [review]
LGTM

File: web/src/features/panels/LeftPanel.tsx (hunk 1 above)
Comment (madsnedergaard) [reply web/src/features/panels/LeftPanel.tsx:38]: Yeah, I thought so too, but couldn't get something smart working - so I just hacked it for now 😅

**Pull request merged**